llm/app/models/reservation.py

```
@@ -0,0 +1,17 @@
+# reservation.py
+from sqlalchemy import Table, Column, Integer, String, Date, Time, ForeignKey, MetaData
```

Ruff (0.12.2)

`sqlalchemy.MetaData` imported but unused

Remove unused import: `sqlalchemy.MetaData`

(F401)